<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="fileView">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>511</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clientSelect">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>70</y>
      <width>79</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_labSelect">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>130</y>
      <width>79</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_addToList">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delItem">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>320</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_compare">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>240</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Compare</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_report">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>320</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Report</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="graphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>511</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_plot">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>770</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>770</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_quit">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>770</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fileList">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File list:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_date">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>190</y>
      <width>79</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QTableView" name="constView">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>400</y>
      <width>431</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_constEdit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>770</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>691</x>
      <y>32</y>
      <width>331</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_ClientFile">
       <property name="text">
        <string>Client file:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_clientFile"/>
     </item>
     <item>
      <widget class="QLabel" name="label_labFile">
       <property name="text">
        <string>Lab file:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_labFile"/>
     </item>
     <item>
      <widget class="QLabel" name="label_date">
       <property name="text">
        <string>Currrent date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_date"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
